File: tests/Generator/resource/kotlin/complex/StringPropertyType.kt

import com.fasterxml.jackson.annotation.*

/**
 * Represents a string value
 */
open class StringPropertyType : ScalarPropertyType {
    @JsonProperty("default")
    var default: String? = null
    @JsonProperty("format")
    var format: String? = null
    @JsonProperty("type")
    var type: String? = "string"
}

File: tests/Generator/resource/swift/complex/StringPropertyType.swift
// Represents a string value
class StringPropertyType: ScalarPropertyType {
    var _default: String?
    var format: String?
    var _type: String? = "string"

    enum CodingKeys: String, CodingKey {
        case _default = "default"
        case format = "format"
        case _type = "type"
    }
}

